components/vars/css/components/spectrum-badge.css

```diff
@@ -1,6 +1,5122 @@
 :root {
-  --spectrum-badge-text-color: var(--spectrum-global-color-static-white);
-  --spectrum-badge-error-background-color: var(--spectrum-semantic-negative-color-default);
-  --spectrum-badge-info-background-color: var(--spectrum-semantic-informative-color-default);
-  --spectrum-badge-warning-background-color: var(--spectrum-semantic-notice-color-default);
+  --spectrum-badge-fuchsia-l-focus-ring-border-radius-key-focus: var(--spectrum-global-dimension-size-75);
```

Comment:
Is it required that the css variables are now a combination of color and size? Was looking at the previous label css and it looked as if there were separate variables for sizing and colors.

Reply:
Hmm, this looks wrong. @GarthDB thoughts?

Reply:
Any updates on this? @GarthDB @lazd

Reply:
@jluyau the CSS vars file has the size and color tokens. What's new in `spectrum-badge` file are the t-shirt sizes. T-shirt sizes are separate from `scale` (i.e., `large`, `medium`). We are going to see a ton of new tokens for each component that gets t-shirt sizing.

Reply:
Thanks for the update @GarthDB